var/spack/repos/builtin/packages/flux-core/package.py

```diff
@@ -48,6 +48,7 @@ class FluxCore(AutotoolsPackage):
     depends_on("py-pyyaml", type=('build', 'run'), when="@0.11.0:,master")
     depends_on("py-jsonschema", type=('build', 'run'), when="@master")
     depends_on("jansson")
+    depends_on("pkg-config")
```

Comment:
Is there a reason why we use `pkg-config` (a *concrete package*) instead of `pkgconfig` (a *virtual dependency* satisfied by both `pkg-config` and `pkgconf`)?

Reply:
> Is there a reason why we use `pkg-config` (a _concrete package_) instead of `pkgconfig` (a _virtual dependency_ satisfied by both `pkg-config` and `pkgconf`)?

@alalazo Thanks for catching this. I didn't realize there was a pkgconfig virtual dependency. Changes committed.

Reply:
I think there was an accidental change on `qt` committed too...